src/lib/daemon_rpcs/types.ml

```diff
@@ -213,18 +213,7 @@ module Status = struct
     let conf_dir = string_entry "Configuration directory"
 
     let peers field =
-      let render display_peer =
-        let open Network_peer.Peer in
-        of_display display_peer |> to_multiaddr_string
-      in
-      map_entry
-        (sprintf "Peers (%d)" (List.length @@ FieldT.get field))
-        ~f:(fun peers ->
-          List.mapi peers ~f:(fun i peer ->
-              let rendered = "\t" ^ render peer in
-              if i = 0 then "\n" ^ rendered else rendered )
-          |> String.concat ~sep:"\n" )
-        field
+      Some ("Peers", sprintf "%d" @@ List.length (FieldT.get field))
```

Comment:
Can you make it optional, toggled by a flag?

Reply:
will do

Reply:
seems too difficult to pass a flag here